src/lib/operators/aggregate_hash.cpp

```diff
@@ -816,15 +821,39 @@ write_aggregate_values(pmr_vector<AggregateType>& values, pmr_vector<bool>& null
 }
 
 void AggregateHash::_write_groupby_output(RowIDPosList& pos_list) {
+  auto& step_performance_data = static_cast<OperatorPerformanceData<OperatorSteps>&>(*performance_data);
+  Timer timer;  // _aggregate above has its own, internal timer. Start measuring once _aggregate is done.
+
   auto input_table = left_input_table();
 
+  auto unaggregated_columns = std::vector<std::pair<ColumnID, ColumnID>>{};
```

Comment:
Is there a reason not calling these `unaggregate_columns` simply `group_by_columns`?

Reply:
Yes. `group_by_columns` is already used for the columns that we actually group on. The fact that the ANY columns used to be part of the group by is nothing that the aggregate operator should care about.